client/src/components/Feed/Post.js

```diff
@@ -338,57 +337,59 @@ const Post = ({
-                user &&
-                (user._id === post.author.id || user.id === post.author.id) && (
-                  <SubMenuButton
-                    onSelect={onSelect}
-                    onChange={onChange}
-                    postId={postId}
-                    post={post}
-                    user={user}
-                  />
-                )}
-            </div>
-          </div>
-          <WhiteSpace size="md" />
-          {renderTags}
-          <WhiteSpace />
-          {renderContent}
-          {fullPostLength > CONTENT_LENGTH && (
-            <RenderViewMore
-              postId={postId}
-              onClick={onClick}
-              loadMorePost={loadMorePost}
-            />
-          )}
-          {renderSocialIcons}
-          {renderShareModal}
-          {renderComments}
-          <WebModal
-            title="Confirm"
-            visible={deleteModalVisibility}
-            onOk={() => postDelete(post)}
-            onCancel={handlePostDelete}
-            okText="Delete"
-            cancelText="Cancel"
+        <div>
+          <PostCard
+            style={{
+              display: "inline-block",
+              maxWidth: "80rem",
+              marginTop: "1rem",
+            }}
           >
-            <p>Are you sure you want to delete the post?</p>
-          </WebModal>
+            <div className="card-header">
+              {renderHeader}
+              <div className="card-submenu">
+                {isAuthenticated &&
+                  user &&
+                  (user._id === post.author.id || user.id === post.author.id) && (
+                    <SubMenuButton
+                      onSelect={onSelect}
+                      onChange={onChange}
+                      postId={postId}
+                      post={post}
+                      user={user}
+                    />
+                  )}
+              </div>
+            </div>
+            <WhiteSpace size="md" />
+            {renderTags}
+            <WhiteSpace />
+            {renderContent}
+            {fullPostLength > CONTENT_LENGTH && (
+              <RenderViewMore
+                postId={postId}
+                onClick={onClick}
+                loadMorePost={loadMorePost}
+              />
+            )}
+            {renderSocialIcons}
+            {renderShareModal}
+            {renderComments}
+            <WebModal
+              title="Confirm"
+              visible={deleteModalVisibility}
+              onOk={() => postDelete(post)}
+              onCancel={handlePostDelete}
+              okText="Delete"
+              cancelText="Cancel"
+            >
+              <p>Are you sure you want to delete the post?</p>
+            </WebModal>
+          </PostCard>
           {showComments && (
-            <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
-          )}
-        </PostCard>
+              <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
```

Comment:
is this some issue with your prettier settings? the formatting seems a bit off here